app-modules/knowledge-base/resources/views/filament/pages/badges.blade.php

```diff
@@ -33,7 +33,36 @@
 --}}
 
 <div class="flex gap-2">
-    <x-filament::badge>
-        {{ $getRecord()?->status?->name ?? '' }}
-    </x-filament::badge>
+    @if ($getRecord()->status)
+        <x-filament::badge>
+            {{ $getRecord()?->status?->name ?? '' }}
+        </x-filament::badge>
+    @endif
+    @if (!empty($getRecord()->public) && !empty($getRecord()->category_id))
+        <x-filament::badge
+            icon="heroicon-m-clipboard"
+            icon-position="after"
+        >
+            <button
```

Comment:
Would it work to move this `button` tag to surround the badge? So the entire badge is clickable?

Reply:
Done